server/index.js

```diff
@@ -31,6 +32,13 @@ app.get('/photos/:propertyID/primaryPhoto', (req, res) => {
     })
 });
 
+  //add one final route to take care of any other endpoints - important to allow sending back of the index.html
+  app.get('*', (req, res) => {
+    //deliver index.html --> this will deliver the bundle too
+    const currPath = path.resolve(__dirname + '/../client/dist/index.html')
+    res.sendFile(currPath);
+    });
+
```

Comment:
This is cool. I think I need to update my service to do the same thing. Does this mean if you visit localhost:3002/2, /3, /4, etc, index.html will be the response?